charts/rancher-webhook/rancher-webhook/0.1.100+up0.1.1/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/hidden: "true"
+  catalog.cattle.io/namespace: cattle-system
+  catalog.cattle.io/os: linux
+  catalog.cattle.io/release-name: rancher-webhook
+apiVersion: v2
+appVersion: 0.1.1
+description: ValidatingAdmissionWebhook for Rancher types
+name: rancher-webhook
+version: 0.1.100+up0.1.1
```

Comment:
Doesn't need a min rancher version as its being introduced here to fix former rancher versions. 
Last version was 0.1.000 so 0.1.100 is correct.